CRM/Contribute/BAO/Contribution.php

```diff
@@ -4022,7 +4011,7 @@ public static function buildOptions($fieldName, $context = NULL, $props = []) {
             'version' => 3,
             'id' => ($props['contribution_page_id']),
           ]);
-          $types = (array) CRM_Utils_Array::value('payment_processor', $page, 0);
+          $types = (array) $page['payment_processor'] ?? 0;
```

Comment:
Need brackets since the cast will apply to the first one first.

Reply:
@demeritcowboy if you want to pull out a handful of these to a reviewer's commit I'm happy to merge them - maybe a half dozen?

Reply:
Ok I pulled out a few - https://github.com/civicrm/civicrm-core/pull/18391.

Reply:
Thanks - I'll close this for now & if anyone wants to use it to help with another round they can (although from past experience doing these is easy - getting them reviewed is hard)